### PR TITLE
feat(): ignoring stateless feature impl

### PR DESCRIPTION
**Pull request by Christiantyemele**

### CODE REVIEW

File: crates/plugins/mediator-coordination/src/web/coord/handler/stateful.rs
Comment (IngridPuppet) [inline crates/plugins/mediator-coordination/src/web/coord/handler/stateful.rs:1]: Can you auto-format this file? The trailing spaces are just noise.

Comment (Christiantyemele) [reply crates/plugins/mediator-coordination/src/web/coord/handler/stateful.rs:1]: auto-formatting find nothing to format,i mean no effect

**Review by Hermann-Core — CHANGES_REQUESTED** [review]
I think you should also set **stateful** feature to the `stateful` module here. Because even if we don't enable the feature, the module will still be built. I don't think that this is the expected behavior.
https://github.com/adorsys/didcomm-mediator-rs/blob/bdc5d36a3d4ab6a75aa500b38be1872bd9e49586/crates/plugins/mediator-coordination/src/model/mod.rs#L1-L5

**Review by IngridPuppet — APPROVED** [review]

**Comment by Christiantyemele** [comment]
> I think you should also set **stateful** feature to the `stateful` module here. Because even if we don't enable the feature, the module will still be built. I don't think that this is the expected behavior.
> 
> https://github.com/adorsys/didcomm-mediator-rs/blob/bdc5d36a3d4ab6a75aa500b38be1872bd9e49586/crates/plugins/mediator-coordination/src/model/mod.rs#L1-L5

Ok

**Review by Hermann-Core — APPROVED** [review]

**Pull request merged**